src/main/kotlin/lotto/view/InputView.kt

```diff
@@ -0,0 +1,37 @@
+package lotto.view
+
+object InputView {
+    fun getPurchaseAmount(): Long {
+        println("구입금액을 입력해 주세요.")
+        return convertToLong(readLine())
+    }
+
+    fun getWinningNumbers(): List<Int> {
+        println("\n지난 주 당첨 번호를 입력해 주세요.")
+        return convertToListInt(readLine())
+    }
+
+    private fun convertToLong(moneyString: String?): Long {
+        moneyString.checkNullOrBlank()
+        return moneyString!!.toNumericLong()
+    }
+
+    private fun convertToListInt(winningNumberStr: String?): List<Int> {
+        winningNumberStr.checkNullOrBlank()
+        return winningNumberStr!!.split(",").map { it.toNumericInt() }
+    }
+}
+
+fun String?.checkNullOrBlank() = require(!this.isNullOrBlank()) { "입력값이 비어있어요." }
+
+fun String.toNumericInt(): Int {
+    val number = this.trim().toIntOrNull()
+    require(null != number && number > 0) { "0보다 큰 숫자만 입력해 주세요. inputMoney: $this" }
+    return number!!
+}
```

Comment:
밑에 `toNumericLong`과 같아보여서 여기에만 코드를 남겨볼게요!
```suggestion
fun String.toNumericInt(): Int {
    val number = this.trim().toIntOrNull() ?: throw IllegalArgumentException("잘못된 숫자 입력입니다.")
    require(number > 0) { "0보다 큰 숫자만 입력해 주세요. inputMoney: $this" }
    
    return number
}
```

Reply:
위와 같이 `?:`를 쓰게 된다면, 메세지도 더 명확하게 나눌 수 있을거 같습니다.

Reply:
오 의도가 더 명확해 보여서 좋네요 감사해요~